.craft.yml

```diff
@@ -38,8 +38,10 @@ targets:
         onlyIfPresent: /^sentry-vue-.*\.tgz$/
       'npm:@sentry/gatsby':
         onlyIfPresent: /^sentry-gatsby-.*\.tgz$/
+      'npm:@sentry/angular-ivy':
+        onlyIfPresent: /^sentry-angular-ivy-.*\.tgz$/
       'npm:@sentry/angular':
-        onlyIfPresent: /^sentry-angular-.*\.tgz$/
+        onlyIfPresent: /^sentry-angular-\d.*\.tgz$/
```

Comment:
Maybe this would be a bit more explicit:

```js
/^sentry-angular-(?!ivy-).*\.tgz$/
```

Reply:
Hmm not sure, this would actually be safer than the `\d` implementation. Cause _theoretically_ (not saying this happens but nevertheless) `sentry-angular-somethingotherthanivy-7.39.0.tgz` would be matched by this regex. 

Anyway, I think we're safe with the two current regexes so I'd keep it this way

Reply:
I mean, yeah, sure, but that also applies to any other package regex we have :D

Maybe it would be better for consistency to then update _all_ regexes to have the `\d` part? My main concern is that it may be unclear in the future why this is slightly different then the others, you copy-paste one of them (which one?) etc.

Reply:
Updated all other packages [a54a553](https://github.com/getsentry/sentry-javascript/pull/7303/commits/a54a553172fdfa265f0898236b724bacc29169f0)